crates/bevy_a11y/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "bevy_a11y"
+version = "0.9.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Try to copy the metadata from the other first party crates into this Cargo.toml.